### ISSUE TITLE
Messes up assignments using the ternary operator

### ISSUE THREAD

**Issue by ShimShamSam** [steve8708/vscode-align#9]
```
var foo = a ? b : c;
var foobar = asd ? 123 : 456
```

Becomes

```
var foo    = a ? b :     c;
var foobar = asd ? 123 : 456;
```

It should stop aligning after the first alignment character (=) instead of aligning ALL characters.

**Comment by jeffcalaway** [comment]
I agree with ShimShamSam. Would love to see this. I use ternaries everywhere and Atom Align handled this nicely.

**Comment by steve8708** [comment]
Hey @heycalaway @ShimShamSam, thank you for the feedback! 

I'm currently short on time to extend this package but pull requests are more than welcome!
